napari/components/viewer_model.py

```diff
@@ -645,7 +645,7 @@ def add_image(
             A vector of shear values for an upper triangular n-D shear matrix.
             If a list then must have same length as the axis that is being
             expanded as channels.
-        affine: n-D array or napari.utils.transforms.Affine
+        affine : n-D array or napari.utils.transforms.Affine
```

Comment:
this seem to be the thing causing problem.

Reply:
and that's because `napari.layers.image.image.Image` doc is wrong:

```
     Either a vector of upper triangular values, or an nD shear matrix with
        ones along the main diagonal.
    affine: n-D array or napari.utils.transforms.Affine
        (N+1, N+1) affine transformation matrix in homogeneous coordinates.
        The first (N, N) entries correspond to a linear transform and
        the final column is a lenght N translation vector and a 1 or a napari
        AffineTransform object. If provided then translate, scale, rotate, and
        shear values are ignored.
    opacity : float
        Opacity of the layer visual, betw
```

see the missing space before the `:` in affine; this make numpydoc believe the the name of the parameter is `affine: n-D array or napari.utils.transforms.Affine` with type `None` instead of 
`affine` with type `n-D array or napari.utils.transforms.Affine`

Reply:
which shows clearly in my new terminal documentation renderer....
<img width="861" alt="Screen Shot 2021-01-28 at 21 56 23" src="https://user-images.githubusercontent.com/335567/106237467-c198f800-61b3-11eb-9bc0-9be8d2d1f4ff.png">

I pushed a new commit that should update the class docstrings.